### ISSUE TITLE
LeNet-5 solution support for TF 1.0

### ISSUE THREAD

**Issue by mkolod** [udacity/sdc-issue-reports#541]

**Comment by mkolod** [comment]
In the Lenet-5 solutions in the CNN lesson, the re is a bit of a problem when running TF 1.0. The line that specifies cross-entropy should actually be

`cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=one_hot_y)`

Not specifying the method arguments by name causes this example not to work in TF 1.0. I believe this is backwards compatible, and so won't break the example for TF 0.12 users, but I haven't tested it with TF 0.12.

**Comment by baumanab** [comment]
Thanks for reporting this.  The line in question has been updated.
